src/components/ContactUsForm/index.jsx

```diff
@@ -29,14 +29,14 @@ const ContactUsForm = () => {
   // encode is used for passing the string to the body
   // setTimeOut stands for showing post submission message about a time after submit
   const handleSubmit = (e) => {
+    setIsLoading(true);
     fetch("/", {
       method: "POST",
       "data-netlify": "true",
       headers: { "Content-Type": "application/x-www-form-urlencoded" },
       body: encodeFormData({ "form-name": "contact", ...formValue }),
     }).then(() => {
-      setTimeout(() => setIsSubmitted(false), 2000);
-      setIsSubmitted(true);
+      setIsLoading(false);
```

Comment:
you still want to show the postsubmission message. you just want to show it after its submitted. so you still want to keep setIsSubmitted(true), you just dont want to set it back to false after. please give that a try, if you understand the process of what your code is doing then it should make sense :) 

* you fetch
* you wait until fetch finishes
* you change things in the page in the .then()

so think about how you can make the submission message show up

Reply:
It seems, I have misunderstood your first comment.

Reply:
For showing postSubmissionMessage, actually I also tried ant design success message via the [link](https://codesandbox.io/s/oq9d1) that I have found in the documentation.I created success function and pass it to onClick at the button and it worked. The problem, it gave an error when I put translation code instead of a string message. The error was "identifier is needed".  I also had the same problem when I was working to show postSubmissionMessage using a function. Since I couldn't solve the problem, that's why I used setIsSubmitted instead.

`const success = () => {
  message.success('This is a success message');
};`

Reply:
I will try to understand the process and make it work.

Reply:
the success message appears when you call `  message.success('This is a success message');`

so the question is: when do we want the success message to appear?

we want the success message to appear when the fetch finishes. which part of the code is when the fetch finishes?

Reply:
`.then()` part the fetch finishes.

Reply:
Post submission message is working right now, PTAL.